ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityTextAnswer.component.ts

```diff
@@ -90,7 +90,9 @@ export class ActivityTextAnswer implements OnChanges, OnInit {
     }
 
     public onChange(value: string): void {
-        const answer = value ? value : null;
+        // turns out server will not accept null for text questions
+        // send emit empty string for falsy values
+        const answer = value ? value : '';
```

Comment:
Actually, answers can be nullable https://github.com/broadinstitute/ddp-study-server/blob/develop/pepper-apis/docs/specification/src/components/schemas.yml#L259

Reply:
The server rejects NULLs for text questions. Tested it out and saw the backing Java code and had discussion with @yufengwng about this. I think rejecting NULLs is not unreasonable, but we should update the API doc.

Reply:
Good to know, thanks.